ui/src/Message.elm

```diff
@@ -146,13 +188,15 @@ view model =
                                     [ Input.text [ Input.id "message", Input.onInput SubmitMessage ]
                                     ]
                                 , Button.button [ Button.secondary, Button.attrs [ Spacing.ml1 ], Button.onClick SendMessage ] [ text "Send" ]
+                                , Button.button [ Button.secondary, Button.attrs [ Spacing.ml1 ], Button.onClick WebsocketSend ] [ text "Websocket" ]
```

Comment:
Why the tet is "Websocket"? It would be better that it is something like "Recieve" or "Listen" - user does not have to know really about underlaying technology.

Reply:
Also, you need to transform this button in something like "Stop" or something, to stop listening (i.e. unsubscribe, i.e. close this websocket),

Reply:
Addressed in the last commit.